lib/fluent/plugin/in_dummy.rb

```diff
@@ -58,12 +60,16 @@ def initialize
     def configure(conf)
       super
       @dummy_index = 0
+      config = conf.elements.select{|e| e.name == 'storage' }.first
+      @storage = storage_create(usage: 'suspend', conf: config, type: :local)
```

Comment:
Rebase this tree on master, and please use `default_type` instead of type keyword argument.